src/Analyzers/CSharp/CodeFixes/UseImplicitOrExplicitType/UseExplicitTypeCodeFixProvider.cs

```diff
@@ -74,42 +80,68 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
             if (declarationContext is VariableDeclarationSyntax varDecl)
             {
                 typeSyntax = varDecl.Type;
+                variableIdentifier = varDecl.Variables.First().Identifier;
```

Comment:
This could be replaced with a .Single() right, because if its' var we can't ever have more than one? Or do we let this be invoked even on invalid code? Can you add a comment to that effect as otherwise the 'first' looks odd here.

Reply:
yeah, i am curious what's up here.